src/components/Text/Text.module.css

```diff
@@ -4,14 +4,6 @@
     # TEXT
 \*------------------------------------*/
 
-/**
- * Remove top and bottom margins applied by browser for text.
- */
-.text {
-  margin-bottom: 0;
-  margin-top: 0;
-}
-
```

Comment:
Preflight does this

Reply:
Yea, also noticed some small discrepancies in spacing in GST such as the missing left padding in the InputField component which this help resolve as well